src/icons/fileIcons.ts

```diff
@@ -797,13 +797,15 @@ export const fileIcons: FileIcons = {
         { name: 'tailwindcss', fileNames: ['tailwind.js', 'tailwind.config.js'] },
         { name: '3d', fileExtensions: ['stl', 'obj'] },
         { name: 'buildkite', fileNames: ['buildkite.yml', 'buildkite.yaml'] },
-        { name: 'netlify', fileNames: ['netlify.toml'] },
+        { name: 'netlify', fileNames: ['netlify.json', 'netlify.yml', 'netlify.yaml', 'netlify.toml', '_headers', '_redirects'] },
```

Comment:
Is the "_headers" folder name only be used for netlify projects? In my opinion it's too general... same for "_redirects". I mean everyone who has a folder named "_redirects" will see the netlify folder icon there? That should not be the case.